.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "microsoft.dotnet.darc": {
+      "version": "1.1.0-beta.22519.1",
```

Comment:
You could use eng/common/darc-init.ps1 for this. That will always pull the latest production deployed version.

Reply:
I was thinking about this and I think that we'd prefer pinned version of the tooling that created the VMR?
If we have a servicing branch of the VMR which is in some shape that the tooling that assembled it at that time understands, we might want to keep it that way?

Stupid example: we decide to rename `source-mappings.json` to something else. I would still expect that old servicing branches of the VMR will keep updating without us having to have to go there and rename it?

Reply:
If we make a change in the tooling that we want to backport, we can always bump in servicing branches?
With pinned version, we will have catalogued the VMR+the tooling it took to update it to that version.

Reply:
Lastly, I think it should behave deterministically - you shouldn't get two different VMR states when running the build at a different point in time.

Reply:
Okay fine by me.